deduplicator/src/tests.rs

```diff
@@ -5,8 +5,10 @@ use std::fs::File;
 use std::io::prelude::*;
 use std::path::PathBuf;
 
+use importer_openaddress::OpenAddress;
 use importer_tools::Address;
 use importer_tools::CompatibleDB;
+use libflate::gzip::Decoder;
```

Comment:
You don't need them if you don't test. Therefore, two options in here: either you only import them on test cfg, or you move your test inside a module with `#[cfg(test)]` on it and make the imports there.